posthog/api/utils.py

```diff
@@ -19,16 +20,37 @@ class PaginationMode(Enum):
 
 
 def get_target_entity(request: request.Request) -> Entity:
```

Comment:
Q: What caused this change? I didn't see anything else relating to this changing.

Reply:
Retrieving groups from trends test was failing because this helper function was building the entity using solely ENTITY_TYPE and ENTITY_MATH but didn't have the group type index